.travis.yml

```diff
@@ -1,5 +1,15 @@
+sudo: required
+dist: trusty
 language: node_js
 node_js:
   - "6"
-script: npm run cover
+before_install:  
```

Comment:
I've added a comment to the travis.yml file. I took it from this blog post: http://blog.500tech.com/setting-up-travis-ci-to-run-tests-on-latest-google-chrome-version/